plasma_framework/contracts/src/framework/ExitGameController.sol

```diff
@@ -170,7 +170,7 @@ contract ExitGameController is ExitGameRegistry {
             queue.delMin();
             processedNum++;
 
-            processor.processExit(exitId, vaultId, token);
+            processor.processExit(exitId, vaultId, token, msg.sender);
```

Comment:
hmm, in another comment I thought it is for front-running, but if it is so, the protection should be in this function instead right?